### PR TITLE
vision based relative positioning

### PR DESCRIPTION
**Pull request by dewagter**
@wv-tud

### CODE REVIEW

**Comment by kirkscheper** [comment]
Its useful if you could add a little description with that is in the pull request or what the added module does for the commit history and pr logs.

Also, don't forget need to run code_style

**Comment by wackv** [comment]
Will do. But won't be for a while due to upcoming flight tests.

**Comment by wackv** [comment]
@benelgiz I haven't gotten around to adding it, I will see if I can do so
asap

File: sw/airborne/modules/computer_vision/active_random_filter.cpp [hunk 1]
```diff
@@ -0,0 +1,1959 @@
...
+ * @file "modules/computer_vision//cv_active_random_filter.c"
+ * @author Wilco Vlenterie (wv-tud)
+ * Active random sampling colour filter
+ */
+
+#include "active_random_filter.h"
+#include <vector>
+#include <ctime>
+
+#define BOARD_CONFIG "boards/bebop.h"       // TODO: WHYYYY?
```
Comment (gautierhattenberger) [inline sw/airborne/modules/computer_vision/active_random_filter.cpp]: good question, why is this needed ?
if should be something like `#include BOARD_CONFIG` in place of the `#include "boards/bebop.h"` below and removing this line

**Comment by kirkscheper** [comment]
I will update this after #2364

**Comment by gautierhattenberger** [comment]
up?
